SPECS/net-snmp/net-snmp.spec

```diff
@@ -19,8 +19,8 @@ BuildRequires:  net-tools
 %endif
 Requires:       systemd
 Requires:       perl(:MODULE_COMPAT_%(eval "`perl -V:version`"; echo $version))
+Requires:       %{name}-libs = %{version}-%{release}
 Provides:       %{name}-utils = %{version}-%{release}
-Provides:       %{name}-libs = %{version}-%{release}
 Provides:       %{name}-agent-libs = %{version}-%{release}
```

Comment:
I haven't noticed this one before, sorry. Since we have all the libs in a separate package now, I guess this needs an update as well:

```suggestion
```